### PR TITLE
[CopyFilesOverSSHV0] Add more logging when connection is failed to close

### PR DESCRIPTION
**Pull request by AlexandrKravchuk**
PR adds more informative messages in case of ssh connecting issues as a general improvement and to help to investigate and fix https://github.com/microsoft/azure-pipelines-tasks/issues/11247

Example trace message:
```
2020-04-23T12:02:41.2400256Z ##[debug]Closing the client connection
2020-04-23T12:02:41.2400783Z ##[debug]Failed to close SFTP client:Error: handle is not a Buffer
2020-04-23T12:02:41.2401260Z ##[debug]Task successfully accomplished
```

### CODE REVIEW

File: Tasks/CopyFilesOverSSHV0/sshhelper.ts [hunk 1]
```diff
@@ -80,7 +80,7 @@ export class SshHelper {
                 this.sftpClient = null;
             }
         } catch(err) {
-            tl.debug('Failed to close SFTP client.');
```
Comment (sgrebnov) [inline Tasks/CopyFilesOverSSHV0/sshhelper.ts:83]: Nit: 
'Failed to close SFTP client: ' + err 
or
 `Failed to close SFTP client: ${err}`

**Comment by sgrebnov** [comment]
Nit: 
Ideally, as a reviewer, I would like to have resultant log message (test run with error simulated) as part of PR to ensure it works/looks good.

File: Tasks/CopyFilesOverSSHV0/sshhelper.ts (hunk 1 above)
Comment (AlexandrKravchuk) [reply Tasks/CopyFilesOverSSHV0/sshhelper.ts:83]: Sounds fair

**Comment by AlexandrKravchuk** [comment]
> Nit:
> Ideally, as a reviewer, I would like to have resultant log message (test run with error simulated) as part of PR to ensure it works/looks good.

2020-04-23T12:02:41.2400256Z ##[debug]Closing the client connection
2020-04-23T12:02:41.2400783Z ##[debug]Failed to close SFTP client:Error: handle is not a Buffer
2020-04-23T12:02:41.2401260Z ##[debug]Task successfully accomplished

**Pull request merged**